Marsiple/Comment/CommentViewController.swift

```diff
@@ -23,14 +25,16 @@ class CommentViewController: UIViewController {
 }
 
 private extension CommentViewController {
-    @objc func saveButtonTapped() { // TODO: Implement posting comment on-line
+    @objc func saveButtonTapped() {
+        let comment = Comment(id: 1, name: "John Doe", email: "johndoe@gmail.com", body: commentView.text, postId: postId) // NOTE: - Id is set to 1, but it will be supplied by API. There is no name and email (possible upgrade when registration is implemented).
```

Comment:
`Id is set to 1, but it will be supplied by API`, actually this is incorrect. If you specify the `id`, the `API` will take the provided `Id` and will not assign it's own. 

To solve this, we should eject the `id` from being included in the `JSON`, but only when a `Comment` instance is `encoded`, since we want to receive the `id` when `decoding` the server response.

Also, lets format this as so:

```
 // NOTE: - Id is set to 1, but it will be supplied by API. There is no name and email (possible upgrade when registration is implemented).
let comment = Comment(id: 1,
                                           name: "John Doe",
                                           email: "johndoe@gmail.com",
                                           body: commentView.text,
                                           postId: postId)
```